partials/_instructions.php

```diff
@@ -247,6 +276,17 @@
         <p><strong>[crouton_map map_search_zoom="10"]</strong> - specifies the starting zoom level on the map (default: 10).</p>
         <p><strong>[crouton_map map_search_width="-50"]</strong> - specifies how many meetings to return, a positive integer means how many miles or kilometers to search.  A negative integer indicates the closest number of meetings from that point with no distance limits. (default: -50 [the fifty closest meetings to the point selected]).</p>
         <p><em>The Google API Key must be entered on the crouton settings page for this to work. You must have the 'Google Maps JavaScript API' enabled on your key. For more information on setting up and configuring a Google Maps API key check out this blog article <a target="_blank" href="https://bmlt.app/google-api-key/">https://bmlt.app/google-api-key/</a></em></p>
+    </div>
+	<h3 class="help-accordian"><strong>Create Meeting-Detail Pages</strong></h3>
+    <div>
+        <p>With this shortcode you can insert a HandlebarJS template into your page. 
+        <p><strong>[bmlt_handlebar]</strong></p>
+        <p> If the page is called with a query string including "meeting-id=<em>your meeting here</em>"
+			the default BMLT root server will be accessed to obtain the meeting information used when evaluation the template.  The Meeting Data template, and all other templates and helpers
```

Comment:
Probably meant to say "evaluating" rather than "evaluation" here?

BTW - very cool feature.